packages/utils/src/internals/open_graph_parser.ts

```diff
@@ -386,10 +386,10 @@ const parseOpenGraphProperty = (property: OpenGraphProperty, $: CheerioAPI): str
  * Easily parse all OpenGraph properties from a page with just a `CheerioAPI` object.
  *
  * @param $ A `CheerioAPI` object
- * @param additionalProperties Any potential additional OpenGraph properties you'd like to be scraped. Currently existing properties are kept up to date.
+ * @param additionalProperties Any potential additional `OpenGraphProperty` items you'd like to be scraped. Currently existing properties are kept up to date.
```

Comment:
this should turn it into clickable link in the API docs (if you want to have it as a codeblock, you can do something like ``{@apilink OpenGraphProperty | `OpenGraphProperty`}``

```suggestion
 * @param additionalProperties Any potential additional {@apilink OpenGraphProperty} items you'd like to be scraped. Currently existing properties are kept up to date.
```